doc/Articles/Guides/Sync/SyncVars.md

```diff
@@ -75,4 +58,61 @@ class Cat : Pet
 
 You can attach the Cat component to your cat prefab, and it will synchronize both it's `name` and `color`.
 
->   **Warning** Both `Cat` and `Pet` should be in the same assembly. If they are in separate assemblies, make sure not to change `name` from inside `Cat` directly, add a method to `Pet` instead. 
+> [!WARNING]
+> Both `Cat` and `Pet` should be in the same assembly. If they are in separate assemblies, make sure not to change `name` from inside `Cat` directly, add a method to `Pet` instead. 
```

Comment:
needs to be confirmed, odds are that this limitation got lifted

Reply:
Still applies.